content/efcore/getting-started/samples/loading-related-data/multiple-include-methods/Program.cs

```diff
@@ -13,9 +13,11 @@ public static void Main()
                 .Include(b => b.Author)
                 .Single(b => b.Id == 1);
 
+			Console.WriteLine("Included author and checkout record(s) with book - \nId: {0}\n Title: {1}\n Genre: {2}\n Publication Year: {3}\nAuthor: {4} {5}", book.Id, book.Title, book.Genre, book.PublicationYear, book.Author.FirstName, book.Author.LastName);
```

Comment:
Use multiple `Console.WriteLine` statements instead of the newlines. It will make the code more readable.